lib/automations/release/branch-create-handler.js

```diff
@@ -328,6 +328,60 @@ const branchHandler = async ( context, octokit, config ) => {
 		issue_number: prCreated.data.number,
 		body: commentBody,
 	} );
+
+	// Get existing milestone
+	const milestone = await getMilestoneByTitle(
+		context,
+		octokit,
+		releaseVersion,
+		'open'
+	);
+	const milestoneNumber = milestone.number;
+	const milestoneTitle = milestone.title
+		.replace( '"', '' )
+		.replace( '[', '' )
+		.replace( ']', '' );
+	// Close existing milestone
+	const milestoneUpdate = await octokit.issues.updateMilestone( {
```

Comment:
What do you think about also updating the `due_on` date during this update, we set the milestone's due date to the ship date when releasing, so this could save us a step.

Reply:
@opr That's a good point. We can indeed do that. However, there may be times when we don't ship the release on the day we create the release branch. It may take a day or two to complete testing and address any blockers. In those cases, the due date would be incorrect.